### DISCUSSION TITLE
推送aria2下载后文件名全部为index.html

### DISCUSSION THREAD

**Discussion by Cypic** [AlistGo/alist#1187] [Q&A]
RT，使用pikpak推送aria2下载时，名称默认为index.html，下载完成后修改后缀后正常使用，确定为源文件，仅在pikpak中遇到，阿里无此问题，另外手动复制aria2地址在下载时正确显示文件名，下载完成后再次变回index.html，想知道为aria2问题还是alist问题
![QQ截图20220601234509](https://user-images.githubusercontent.com/42772515/171446165-457bd47a-3eac-41f5-91a6-8aee8a6f9141.png)
![QQ截图20220601233939](https://user-images.githubusercontent.com/42772515/171446189-54f45d65-95b1-4207-967e-9f307fe29d8c.png)
![0VEQ7Q~{0( E~H~NCOUU_P](https://user-images.githubusercontent.com/42772515/171446276-5fd9a108-1cee-4b59-bf75-9f67c122b2e2.png)

**Comment by xhofe** [comment]
都不是 pikpak的问题

**Reply by Cypic** [reply]
谢谢回复，那估计暂时无解了，我还是下载手动改名吧
